src/node.cc

```diff
@@ -4473,6 +4492,11 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
   if (debug_enabled)
     EnableDebug(&env);
 
+  if (load_napi_modules) {
+    ProcessEmitWarning(&env, "N-API is an experimental feature "
+        "and could change at any time.");
+  }
+
```

Comment:
CTC has specifically requested command-line enablement along with a warning message for features with experimental status. Tracing does the same thing.